### ISSUE TITLE
For a customer, Get-AzureRmEnvironment cmdlet returning empty result

### ISSUE THREAD

**Issue by Ajay-MS** [Azure/azure-powershell#6017]
### Description

It's specific to the customer, _Get-AzureRmEnvironment_ cmdlet is returning an empty result.

### Script/Steps for Reproduction

* Customer installed Azure PS 5.6 using MSI 
* Executed cmdlet Get-AzureRmEnvironment and cmdlet returning empty result. 

```powershell
Get-AzureRmEnvironment
```

### Module Version

Azure PS version 5.6

```powershell
Get-Module -Name AzureRM -ListAvailable
```

### Environment Data

PS version 5.1

```powershell
$PSVersionTable
```

### Debug Output

Debug output in this case is returning nothting.

**Comment by markcowl** [comment]
@Ajay-MS Please provide the output of 'Get-Module -ListAvailable' on the machien that has the issue.  Also, what cmdlets were executed before this cmdlet, and, if available, what is the content of ```\Users\<username>\AppData\Roaming\Windows Azure PowerShell\AzureRMContext.json```

**Comment by markcowl** [comment]
This was fixed in version 6.0.1 of AzureRM - please have the customer update
